### PR TITLE
Move build scripts from old repo and update them

### PR DESCRIPTION
**Pull request by LefterisJP**
- Have a special shell file to create an environment similar to what
  vsvars32.bat would create for our windows buildbot
- Updated repositories list to reflect the new split repos configuration
- Added an option to clone from the ssh url instead of the https
  one. HTTPS is still the default.
- Just like before both scripts require you to add a project argument in
  order to know which repositories to clone/update/build and in which
  order.

### CODE REVIEW

File: cmake/scripts/ethupdate.sh [hunk 1]
```diff
@@ -5,20 +5,81 @@
...
+	   "mix:https://github.com/ethereum/mix")
+REPOS_SSH_MAP=("webthree-helpers:git@github.com:ethereum/webthree-helpers.git"
+	   "tests:git@github.com:ethereum/tests.git"
+	   "libweb3core:git@github.com:ethereum/libweb3core.git"
+	   "libethereum:git@github.com:ethereum/libethereum.git"
+	   "libwhisper:git@github.com:ethereum/libwhisper.git"
+	   "webthree:git@github.com:ethereum/webthree.git"
+	   "solidity:git@github.com:ethereum/solidity.git"
+	   "alethzero:git@github.com:ethereum/alethzero.git"
+	   "mix:git@github.com:ethereum/mix.git")
```
Comment (debris) [inline cmake/scripts/ethupdate.sh]: I think that you also need to pull `web.js` submodule to build webthree (eth), alethzero and mix

File: cmake/scripts/ethupdate.sh [hunk 2]
```diff
@@ -63,49 +132,77 @@ do
...
-		cd $ROOT_DIR
-		continue
+		if [[ $REQUESTED_PROJECT == "" ]]; then
+			echo "ETHUPDATE - INFO: Skipping ${repository} because directory does not exit";
+			cd $ROOT_DIR
+			continue
+		else
+			echo "ETHUPDATE - INFO: Repository ${repository} for requested project ${REQUESTED_PROJECT} did not exist. Cloning ..."
+			get_repo_url $repository
+			git clone $REPO_URL
```
Comment (debris) [inline cmake/scripts/ethupdate.sh]: if we do not need history, you can add `--depth=1` param. I will be a little faster :)

Comment (LefterisJP) [inline cmake/scripts/ethupdate.sh]: Coudl be an option of the script for the build bots, but these scripts are also meant to be used by us, the developers in which case I want the history to be there

**Comment by debris** [comment]
:+1:

**Pull request merged**